src/quantum/azext_quantum/_params.py

```diff
@@ -19,6 +20,7 @@ def load_arguments(self, _):
 
     with self.argument_context('quantum workspace') as c:
         c.argument('workspace_name', workspace_name_type)
+        c.argument('storage_account', storage_account_name_type)
```

Comment:
It is sort of unfortunate that there is another parameter called `storage` which means something totally different. Not sure there's much that can be done though.

Reply:
Yes, I consider reusing storage for this, but this seemed dangerous, since it has its own purpose.